packages/cli/src/commands/search/dump.ts

```diff
@@ -0,0 +1,169 @@
+import {Command, flags} from '@oclif/command';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+import {
+  Preconditions,
+  IsAuthenticated,
+} from '../../lib/decorators/preconditions/';
+import {Config} from '../../lib/config/config';
+import PlatformClient from '@coveord/platform-client';
+import {writeFile} from 'fs-extra';
+import {Parser} from 'json2csv';
+import {cli} from 'cli-ux';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../hooks/analytics/analytics';
+
+interface SearchResult {
+  raw: {rowid: string};
```

Comment:
I think the `rowid` come from the backend, right?
If not, I'd say `rowId` (i.e. camelCase)